### ISSUE TITLE
Test set unavailability should be solved by train test split on the development set

### ISSUE THREAD

**Issue by manumacc** [manuelemacchia/hotel-sentiment-analysis#1]
The test set is unavailable. Therefore, we should use the development set for training, testing and validation.

To do this, we should split the development set into training and test sets (80-20?). To validate the model, we should perform k-fold cross-validation on the training set. Testing should be done on the test set.
